common/test/acceptance/pages/studio/overview.py

```diff
@@ -600,6 +602,14 @@ def make_exam_timed(self, hide_after_due=False):
         self.q(css=".action-save").first.click()
         self.wait_for_ajax()
 
+    def make_subsection_hidden_after_due_date(self):
+        """
+        Sets a subsection to be hidden after due date.
+        """
+        self.q(css='input[value=hide_after_due]').first.click()
+        self.q(css=".action-save").first.click()
+        self.wait_for_ajax()
```

Comment:
Per the method above this one, and https://openedx.atlassian.net/wiki/pages/viewpage.action?pageId=39583991#Whyismybokchoytestflaky?-Awebpagedidanajaxrequestwhoseresponseshouldbeawaited I used `wait_for_ajax()`. Is the advantage to waiting on a specific element that it is potentially faster?